native_words.asm

```diff
@@ -341,7 +341,7 @@ _not_zero:
                 ; time. This bit will be cleared on the first CTRL-n or CTRL-p
                 ; received and won't be used to calculate the history buffer
                 ; offset.
-                ora #$08
+                ora #$8
```

Comment:
I'm curious here - why take out the leading 0 on #$08?  It's being used as a bit mask, so it's important that the high nybble bits are zero and it shows (with the 2-digit hex value) that an 8-bit value is being used here.

Reply:
yup, that one could be reverted.  i was trying to distinguish magic numbers from bitmasks from small constants like `#3`.   for more complex bitmasks I usually write as binary %00001000 but arguable - all compiles to the same thing after all :)

Reply:
I'm fine with single digit for small values, but this one is a bitmask.  I'd also be fine with the binary representation.  Most assemblers support the % syntax for those.